brush/src/components/Map/style.css

```diff
@@ -0,0 +1,3 @@
+.canvasWrapper {
```

Comment:
Since we are using `webgl`, I think that would be better to call it as `webglWrapper`